tests/MavenTests/common.cpp

```diff
@@ -83,7 +83,7 @@ vector<PeakGroup> common::getGroupsFromProcessCompounds(){
     mavenparameters->clsf = clsf;
     mavenparameters->ionizationMode = -1;
     mavenparameters->matchRtFlag = true;
-    mavenparameters->compoundRTWindow = 2;
+    mavenparameters->compoundRTWindow = 1;
```

Comment:
Cannot see this change reflected in the UI. default_settings.xml needs to be regenerated and pushed. @rish9511 will know more about this

Reply:
I was not sure what this change is actually doing and I am still not. 
In the above changes, Qt settings has been changed to hold '1' for compoundPPMWindow. Well, I did not bother about it as QtSettings no more hold PeakDetection Dialog settings  and here the change has been made in Tests module which is independent of UI.

Anyway, @IpankajI  if you want this to be reflected in the UI then you will have to make changes to default_settings.xml and settings.h .If you not sure about how to do that, ask me or @shubhra-agrawal